flocker/apiclient/_client.py

```diff
@@ -186,6 +247,34 @@ def synchronize_state(self):
                 path=FilePath(b"/flocker").child(bytes(dataset.dataset_id)))
             for dataset in self._configured_datasets.values()]
 
+    def acquire_lease(self, dataset_id, node_uuid, expires):
+        try:
+            self._leases = self._leases.acquire(
+                self._NOW, dataset_id, node_uuid, expires)
+        except LeaseError:
+            return fail(LeaseAlreadyHeld())
+        return succeed(
+            Lease(dataset_id=dataset_id, node_uuid=node_uuid, expires=expires))
+
+    def release_lease(self, dataset_id):
+        # We don't handle the case where lease doesn't exist yet, since
+        # it's not clear that's necessary yet. If it is we'll need to
+        # expand this logic.
+        lease = self._leases[dataset_id]
+        self._leases = self._leases.release(dataset_id, lease.node_id)
+        return succeed(
+            Lease(dataset_id=dataset_id, node_uuid=lease.node_id,
+                  # Just lie about expiration time, it doesn't really
+                  # matter when releasing:
```

Comment:
If it doesn't matter, that suggest suggests the client (and the REST API?) shouldn't bother including it either. But, since it is easy enough to get here, we might as well populate it correctly.